migrations/v7.3.2_8.0.0.js

```diff
@@ -0,0 +1,48 @@
+var mongoose = require('mongoose');
+var _ = require('lodash');
+var es = require('event-stream');
+var config = require('../lib/config/config');
+
+require('../lib/config/mongoose')(mongoose, config);
+
+var Situation = mongoose.model('Situation');
+
+var mapping = {
+    'primeActivite': 'ppa',
+    'prepare': 'paje_prepare',
+    'clca': 'paje_clca'
+};
+
+var keys = _.keys(mapping);
+
+Situation.find({ 'individus.ressources.type': { '$in': _.keys(mapping) } }).stream()
```

Comment:
Good point. I will wait for https://github.com/sgmap/mes-aides-api/issues/123 to be discussed so that migrations are clearer and cleaner.